Userland/Libraries/LibAccelGfx/Painter.cpp

```diff
@@ -417,6 +417,16 @@ void Painter::restore()
     m_state_stack.take_last();
 }
 
+void Painter::set_clip_rect(Gfx::IntRect rect)
+{
+    GL::scissor(rect);
+}
+
+void Painter::clear_clip_rect()
+{
+    GL::scissor({ { 0, 0 }, m_target_bitmap->size() });
```

Comment:
Would it be more idiomatic to disable the scissor test here?

Reply:
yes, I made it to disable scissor test instead.